projects/orquestra-workflow-runtime/Makefile

```diff
@@ -29,6 +29,15 @@ test:
 MIN_COVERAGE=75
 
 
+# We do not want to keep the IRs in json form in the repo as maintanance is a big hussle
+# However we want to make sure that during main testing we do not have SDK installed
+pre-coverage:
+	$(PYTHON) -m pip install -e '../orquestra-sdk'
+	$(PYTHON) -m pip install -e '../orquestra-workflow-shared'
+	cd tests/data; $(PYTHON) generate_workflow_defs.py
+	$(PYTHON) -m pip uninstall -y orquestra-sdk
```

Comment:
Have you considered making this part of `make github_actions` in `orquestra-workflow-runtime`?

Reply:
Yes, I just believe that the separation of steps between:
"Here we prepare our tests"
"Here we run our tests" 
is good practice, thus 2 steps are better here IMHO